flake.nix

```diff
@@ -2,7 +2,13 @@
   description = "Roc flake";
 
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs?rev=fd281bd6b7d3e32ddfa399853946f782553163b5";
+    nixpkgs.url = "github:nixos/nixpkgs?rev=63dacb46bf939521bdc93981b4cbb7ecb58427a0";
+
+    zig-overlay.url = "github:mitchellh/zig-overlay?rev=c7615d701b2a5355a3f3013918b6b945c394e791";
+    zig-overlay.inputs.nixpkgs.follows = "nixpkgs";
+
+    zls-overlay.url = "github:zigtools/zls?rev=a26718049a8657d4da04c331aeced1697bc7652b";
```

Comment:
Not sure if we needed zls as well. I figured I'd leave it here and we can remove if not needed. My thinking was that it might make things easier for development -- but I don't think zed is picking it up correctly for me.

Reply:
Removed, this comment no longer relevant